src/Features/CSharp/Portable/UseLocalFunction/CSharpUseLocalFunctionCodeFixProvider.cs

```diff
@@ -75,20 +75,26 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             var root = editor.OriginalRoot;
             var currentRoot = root.TrackNodes(nodesToTrack);
 
+            var options = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
+            var languageVersion = ((CSharpParseOptions)semanticModel.SyntaxTree.Options).LanguageVersion;
+            var makeStaticIfPossible = languageVersion >= LanguageVersion.CSharp8 &&
+                options.GetOption(CSharpCodeStyleOptions.PreferStaticLocalFunction).Value;
```

Comment:
>PreferStaticLocalFunction [](start = 57, length = 25)

I probably wouldn't have added an option for this. `static` is better :-P

Reply:
i'm worried about any option people can't turn off.  If this starts just telling them to change, and they prefer the briefer form, i don't want them to be unhappy :)